### ISSUE TITLE
5.1: Alias for old name for `<Model>CountOutputTypeDefaultArgs` does not exist

### ISSUE THREAD

**Issue by SevInf** [prisma/prisma#20614]
In 5.1, we renamed `<Model>Args` type to `<Model>DefaultArgs`, while still adding an alias for orginal field if it does not conflict with other models. `*CountOutputTypes` though did not get an alias in any case.

Schema:

```prisma
model Group {
  id    String @id @default(uuid())
  users User[]
}

model User {
  id      String @id @default(uuid())
  name    String
  group   Group  @relation(fields: [groupId], references: [id])
  groupId String
}

```

Code:

```ts
import { Prisma } from '@prisma/client'

declare const x: Prisma.GroupCountOutputTypeArgs
```

This works in 5.0, but breaks in 5.1. 

We should add `type GroupCountOutputTypeArgs = GroupCountOutputTypeDefaultArgs` alias into generated client in the same way we already do for other `Args` types.

**Comment by ymc9** [comment]
Do you think this can be included in Prisma 5.1.2?

**Comment by janpio** [comment]
We are not doing a patch release for this, but it will be included in 5.2.0 which will be released in 1.5 weeks, 2023-08-22.

I understand this is unfortunate for you and your custom generator @ymc9, but a patch release means that _all_ our users that are on the newest version have to do additional work of upgrading (or not "have to", but they tend to) and this bug is not urgent enough to justify that. If you have an issue where your users are asking for a fix for this, please let me know and I can write a message from Prisma there as well.

**Comment by ymc9** [comment]
Thanks for the update @janpio . Having it fixed in the next minor release sounds good!
